velox/common/file/FileSystems.cpp

```diff
@@ -61,6 +61,16 @@ std::shared_ptr<FileSystem> getFileSystem(
   VELOX_FAIL("No registered file system matched with file path '{}'", filePath);
 }
 
+std::vector<std::shared_ptr<filesystems::FileSystem>>
+getRegisteredFileSystems() {
```

Comment:
Unrelated change?

Reply:
This is used here: https://github.com/prestodb/presto/pull/20457/files#diff-baa23cbd705092e9a2cc187cc5ca1237bd72734427b5ef7afb4e94ace27cd2e7R331-R336